### PR TITLE
Add WhatsApp message title field to export and import processes

### PR DESCRIPTION
**Pull request by erikh360**
## Purpose
_Why does this exist?_

## Solution
_What changed, and does this address the problem? _

#### Checklist
- [ ] Added or updated unit tests
- [ ] Added to release notes
- [ ] Updated readme/documentation (if necessary)
- [ ] Add support to FakeCMS in the [flow tester](https://github.com/praekeltfoundation/flow_tester) (if necessary)

### CODE REVIEW

File: home/tests/test_api.py [hunk 1]
```diff
@@ -655,6 +655,7 @@ def test_detail_view_whatsapp_message(self, uclient: Any) -> None:
         assert body["total_messages"] == 1
         assert body["text"]["type"] == "Whatsapp_Message"
         assert body["text"]["value"]["message"] == "*Default whatsapp Content 1* 🏥"
+        assert body["text"]["value"].get("title", "") == ""
```
Comment (jerith) [inline home/tests/test_api.py:658]: If we expect this field to always be in the response, we should index it directly like we do with the other fields.

File: home/tests/test_content_import_export.py [hunk 2]
```diff
@@ -85,6 +85,7 @@ def add_new_fields(entry: ExpDict) -> ExpDict:
     return {
         "whatsapp_template_slug": "",
         **entry,
+        "whatsapp_message_title": entry.get("whatsapp_message_title") or "",
```
Comment (jerith) [inline home/tests/test_content_import_export.py:88]:
```suggestion
        "whatsapp_message_title": entry.get("whatsapp_message_title", ""),
```

File: home/tests/test_page_builder.py [hunk 3]
```diff
@@ -109,6 +109,7 @@ def test_build_simple_pages() -> None:
         (
             "Whatsapp_Message",
             {
+                "title": "",
```
Comment (jerith) [inline home/tests/test_page_builder.py:112]: Do we need a test with a non-empty `title`?

File: home/tests/test_serializers.py [hunk 4]
```diff
@@ -468,6 +468,7 @@ def test_happy_case_single_variation_message(self) -> None:
         expected_result = {
             "type": "Whatsapp_Message",
             "value": {
+                "title": "",
```
Comment (jerith) [inline home/tests/test_serializers.py:471]: Do we need a test with a non-empty `title`?

File: home/export_content_pages.py [hunk 5]
```diff
@@ -126,6 +127,8 @@ def add_message_fields(self, msg_blocks: MsgBlocks) -> None:
                 self.whatsapp_template_slug = whatsapp.value.slug
             else:
                 self.whatsapp_body = whatsapp.value["message"].strip()
+                # Per-message WhatsApp title (not the page-level whatsapp_title)
+                self.whatsapp_message_title = whatsapp.value.get("title", "")
```
Comment (jerith) [inline home/export_content_pages.py:131]: Since `title` should always exist, I think we want `value["title"]` here.

File: home/models.py [hunk 6]
```diff
@@ -1625,6 +1631,12 @@ def __str__(self) -> str:
         """String repr of this snippet."""
         return self.slug
 
+    # Provide a minimal snippet_viewset interface for chooser widgets when the
+    # snippet isn't registered in the admin (feature-flagged). Wagtail's
+    # SnippetChooserBlock accesses `target_model.snippet_viewset.icon`.
+    class snippet_viewset:
+        icon = "order"
+
```
Comment (jerith) [inline home/models.py:1639]: Is this related to the WA title?
